airbyte-integrations/connectors/destination-s3/src/test/java/io/airbyte/integrations/destination/s3/parquet/ParquetSerializedBufferTest.java

```diff
@@ -77,6 +78,51 @@ public void testCompressedParquetWriter() throws Exception {
     runTest(195L, 215L, config, getExpectedString());
   }
 
+  private static String resolveArchitecture() {
+    return System.getProperty("os.name").replace(' ', '_') + "-" + System.getProperty("os.arch") + "-" + System.getProperty("sun.arch.data.model");
+  }
+
+  @Test
+  public void testLzoCompressedParquet() throws Exception {
```

Comment:
what is this testing?

Reply:
> what is this testing?
this unit test emulates the same behavior as described in the dockerfile

Reply:
> do we need to install these dependencies in the test?

unit tests does not use builded destination-s3 image